### PR TITLE
Write healpix

### PR DESCRIPTION
**Pull request by aelanman**
Adds a method to write a HEALPix + frequency sky model to the HDF5 format. This is necessary for the benchmarking branch on `pyuvsim`.

Also cleans up the HEALPix reader tests a little.

### CODE REVIEW

**Comment by aelanman** [comment]
@bhazelton This should be all set now.

File: pyradiosky/tests/test_skymodel.py [hunk 1]
```diff
@@ -902,63 +710,3 @@ def test_array_to_skymodel_loop():
 
     assert np.allclose((sky.ra - sky2.ra).rad, 0.0)
     assert np.allclose((sky.dec - sky2.dec).rad, 0.0)
-
-
-class TestMoon:
```
Comment (bhazelton) [inline pyradiosky/tests/test_skymodel.py]: Why did you remove this test class?

Comment (aelanman) [reply pyradiosky/tests/test_skymodel.py]: Not sure how that happened... it's back now.

File: pyradiosky/skymodel.py [hunk 2]
```diff
@@ -496,6 +496,88 @@ def read_healpix_hdf5(hdf5_filename):
...
+    history: str
+        Optional history string to include in the file.
+
+    """
+    try:
+        import astropy_healpix
+    except ImportError as e:
+        raise ImportError(
+            "The astropy-healpix module must be installed to use HEALPix methods"
+        ) from e
```
Comment (bhazelton) [inline pyradiosky/skymodel.py]: go ahead and stick a `# pragma: no cover` on this block, even when we get the minimum dependency builds working, we won't cover this because we'll skip the test entirely if `astropy-healpix` isn't installed.

Comment (aelanman) [reply pyradiosky/skymodel.py]: This is covered by the new test.
